### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by sjaeiou** [cycdpo/weixin-share#1]
Hi,
Just from yesterday.

This is from my console:

> Sat Feb 10 2018 01:38:30 GMT+0800 (中国标准时间) wx.config begin
> VM597041:1
> {appId: "wxb6b88f29b866f7b3", jsApiList: Array(5), nonceStr: "xs8iricGbVejCTlG", signature: "a996bbf92e4ea946507be9bd467f26ec331127f1", timestamp: "1518197911"}
> Sat Feb 10 2018 01:38:31 GMT+0800 (中国标准时间) wx.config end
> VM597041:1 
> {errMsg: "config:ok", jsApiList: Array(0)}

And if I test a share:

> Sat Feb 10 2018 01:38:36 GMT+0800 (中国标准时间) wx.onMenuShareAppMessage end
> VM597041:1 
> {errMsg: "onMenuShareAppMessage:fail, the permission value is offline verifying"}

Do you know why? It seems that before yesterday this code was working fine. I started to get this error from yesterday.......

**Comment by sjaeiou** [comment]
I understand that if I get "the permission value is offline verifying", which means jsApiList is not configged. But you did add jsApiList into config, the Array just returns a 0 length when "wx.config end"

**Comment by cycjimmy** [comment]
If you do not set `jsApiList` in the parameters of `new WxShare().Config()`. It defaults to `jsApiList: ['onMenuShareTimeline', 'onMenuShareAppMessage', 'onMenuShareQQ', 'onMenuShareQZone', 'onMenuShareWeibo’]`.
And you can override the default lists.

From your console. There is really no configuration jsApiList parameters. You can try to manually add it in `config()`.

**Comment by cycjimmy** [comment]
If you do not set `jsApiList` in the parameters of `new WxShare().Config()`. It defaults to `jsApiList: ['onMenuShareTimeline', 'onMenuShareAppMessage', 'onMenuShareQQ', 'onMenuShareQZone', 'onMenuShareWeibo’]`.
And you can override the default lists.

From your console. There is really no configuration jsApiList parameters. You can try to manually add it in `config()`.

**Comment by sjaeiou** [comment]
Hi, I think I have solved this issue. Your code is good. It is just the domain issue in the public account. 
We set a first level domain to use the JSSDK, but we put this H5 page under a second level domain. It worked fine until yesterday. After tested a whole day. Just figure out that now you must set the exact second level domain which are using this SDK. Guess Wechat has updated their security strategy.
Thank you!
